superfluid.extended.tokenlist.json

```diff
@@ -3,7 +3,7 @@
   "version": {
     "major": 2,
     "minor": 0,
```

Comment:
Shouldn't the major version be 3 here too?

Reply:
nope, that's the package, it can have changes without changing the tokenlist.